deploy/OCPBUGS-1341-PodNetworkConnectivityCheck-leak/config.yaml

```diff
@@ -0,0 +1,13 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  resourceApplyMode: Sync
+  matchExpressions:
+  - key: api.openshift.com/environment
+    operator: In
+    values: ["staging", "integration"]
```

Comment:
This block can probably be removed, we don't need to isolate it to staging/int environments.

Reply:
Chatting out of band, we're ok keeping this limit while we soak in stage/int before promoting to prod.

But we do want to drop the FedRAMP filter, to allow this to run there too, so I've pushed that with 6c4c7a0 -> c5bd4c0.